README.md

```diff
@@ -41,17 +41,14 @@ Part of [GameCI](https://game.ci).
 
 | Description             | Done | Status |
 |-------------------------|------|--------|
-| [Build for WebGL](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#webgl) |
-| [Build for Windows](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#windows) |
-| [Build for Linux](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#linux) |
-| [Build for MacOS](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#macos) |
-| [Build for Android](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#android) |
-| [Build for iOS](https://github.com/marketplace/actions/unity-builder) | ✔ | [![Actions status](https://github.com/game-ci/unity-builder/workflows/Builds/badge.svg?event=push&branch=main)](https://github.com/game-ci/unity-builder#ios) |
-| [Build for Windows store](https://github.com/marketplace/actions/unity-builder) | ❌ | In progress |
-| [Build for tvOS](https://github.com/marketplace/actions/unity-builder) | ❌ | In progress |
-| [Build for PS4](https://github.com/marketplace/actions/unity-builder) | ❌ | Requires license |
-| [Build for XboxOne](https://github.com/marketplace/actions/unity-builder) | ❌ | Requires license |
-| [Build for Switch](https://github.com/marketplace/actions/unity-builder) | ❌ | Requires license |
```

Comment:
Lets leave this in to be as complete as possible.

These badges are a big reason people star this repo. 

I empirically found out through making it much more concise before and people clearly found it less useful, meaning red crosses are very informational and important too. Reverting it increased the amount of people starring the repo by a few hundred percent again.

Reply:
> Lets leave this in to be as complete as possible.

I really doubt anybody is interested in developing for last-gen hardware, so how about we replace Xbox One and PS4 with Xbox Series X and PS5, then?

Reply:
Right. Yes, or just more generic: Xbox, Playstation without a specific version of their platforms.

Reply:
Ok, replaced with the more Generic Xbox and Playstation.